js/init.js

```diff
@@ -30,7 +15,7 @@ function tonicInitDarkmode() {
   }
 }
 
-ginInitDarkmode();
```

Comment:
Darkmode might work after changing this. But you're also more than welcome to keep the name «Gin».